textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/GramBooleanQuery.java

```diff
@@ -18,10 +18,10 @@
 		OR
 	}
 	QueryOp operator;
-	List<String> operandList;
-	List<GramBooleanQuery> subQueryList;
+	Set<String> operandSet;
+	Set<GramBooleanQuery> subQuerySet;
 	
-	int gramLength;
+	static int gramLength;
```

Comment:
@laisycs I think we can change static back to non-static. Here's my thoughts:
`RegexToGramQueryTranslator.translate()` doesn't need to take `gramLength` as a parameter. There are a few places where we need replace 3 with `gramLength`. In all those places, we have the `RegexInfo` object which contains `match`. So we can write `info.match.gramLength`.